Update performance-overhead.mdx (#7479)
getsentry/sentry-docs: src/docs/product/session-replay/performance-overhead.mdx

<Include name="feature-stage-beta-session-replay.mdx" />

Session Replay works by observing and recording changes to your web application's DOM and transmitting that data to event ingestion servers over the public internet. In order to perform this work without negatively impacting the host page's performance, the Session Replay SDK takes care to introduce minimal additional filesize, observe and record DOM content in a non-intrusive way, send the absolute minimum number of bytes required, and use low-latency ingestion endpoints geographically close to your end-users.

**For most web applications, the performance overhead of our client SDK will be imperceptible to end-users.**

> But because performance overhead depends on the complexity of your application, applications with a large DOM and numerous DOM node mutations will have a higher overhead than simpler, mostly static site. The only way to get accurate metrics is to measure performance overhead yourself.

Author: ```suggestion

```

  - The SDK doesn't cause the browser to download and transmit other static assets, like images, video, etc. Instead during playback, those assets are fetched directly from the original host server.
- Sentry’s event ingestion infrastructure uses [distributed Points-of-Presence (PoPs)](https://blog.sentry.io/2022/03/31/sentry-points-of-presence-how-we-built-a-distributed-ingestion/) which place ingestion servers around the world and close to your users. When a Session Replay event is transmitted, the user’s browser connects and transmits the event payload to the closest PoP available for their region. This greatly reduces end-to-end latency and minimizes the amount of networking overhead placed on the browser.
- The SDK is built to gracefully downgrade when needed. In the event several thousand DOM mutations happen at once, the SDK will disable recording to make sure recording the website doesn't further degrade user experience. The [mutation limit values are configurable](/platforms/javascript/session-replay/configuration/#mutation-limits).

## Performance Overhead in Sentry

> 

Author: ```suggestion
While the performance overhead of our client SDK will be imperceptible to end-users most of the time, it can vary based on the complexity of your application. If an applications has a large DOM and numerous DOM node mutations, it will have a higher overhead than simpler, mostly static sites. The only way to get accurate metrics is to measure performance overhead yourself.
```

The benchmark test was a strenuous recording scenario (the Discover data table is one of our most complex, in regards to DOM nodes and mutations). A simpler scenario consisting of navigation to four different "Settings" pages was also ran and produced an increase of ~100 ms of total JS blocking time.

<Note>

>   There's a known performance issue that happens when replays with many DOM

Reviewer: Pedantic i know. There's a few more spot on this page where the apostrophe is  `'` and not `’`.

```suggestion
  There’s a known performance issue that happens when replays with many DOM
```

Author: No, totally appreciate it! Details matter :)
